### PR TITLE
feat/COM130 rewrite standard about using Git

### PR DESCRIPTION
**Pull request by SireeKoolenWijkstra**
Git Flow is new work flow standard
Develop should be default branch
Naming standard for branches introduced
Commits always have to have a title and a description.

### CODE REVIEW

File: general/using-git.md [hunk 1]
```diff
@@ -1,80 +1,64 @@
...
+    - [ ] Enable "Require a pull request before merging"
+    - [ ] Enable "Require approvals"
+    - [ ] Set "Required number of approvals before merging" to at least 1. 
+  - [ ]  Create a new branch every time you're implementing a feature, bug fix or other task.
+  - [ ] Test before you push.
+
+### branches
+- [ ] Branch names must must 
+  - [ ] prefix with either 
+    - [ ] feat
```
Comment (thomasm0) [inline general/using-git.md:26]: This should be `feature`

File: general/using-git.md [hunk 2]
```diff
@@ -1,80 +1,64 @@
...
+    - [ ] chore
+    - [ ] bugfix
+    - [ ] hotfix
+    - [ ] docs
+  - [ ] contain a ticket number that references PBI (product backlog item) if applicable
```
Comment (thomasm0) [inline general/using-git.md:31]: `that references PBI` --> `that references the PBI`

File: general/using-git.md [hunk 3]
```diff
@@ -1,80 +1,64 @@
 # Using Git
-Also see [storing source code](storing-source-code.md).
+> This page was last reviewed May 14th, 2024. It needs to be reviewed again on February 14th, 2025.
+
+## What is the standard for using Git?
+The city of Amsterdam uses Git to push its code to GitHub.
+The city of Amsterdam maintains minimal requirements for work flow, branches and commits.
```
Comment (thomasm0) [inline general/using-git.md:6]: `minimal requirements for work flow` --> `minimal requirements for the workflow`

**Review by thomasm0 — APPROVED** [review]
3 really minor comments, other than that looks good

File: general/using-git.md (hunk 1 above)
Comment (SireeKoolenWijkstra) [reply general/using-git.md:26]: changed it

File: general/using-git.md (hunk 2 above)
Comment (SireeKoolenWijkstra) [reply general/using-git.md:31]: changed it

File: general/using-git.md (hunk 3 above)
Comment (SireeKoolenWijkstra) [reply general/using-git.md:6]: changed it

**Pull request merged**